share/man/man9/cdefs.9

```diff
@@ -0,0 +1,297 @@
+.\"-
+.\" Copyright (c) 2024 M. Warner Losh <imp@FreeBSD.org>
+.\"
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.Dd July 3, 2024
+.Dt CDEFS 9
+.Os
+.Sh NAME
+.Nm cdefs
+.Nd "Compiler portability definitions"
+.Sh DESCRIPTION
+.In sys/cdefs.h
+defines macros for compiler, C language standard portability, POSIX standards
+compliance and symbol visibility.
+It defines programming interfaces for the system header files to adopt to the
+many environments
+.Fx
+supports compilation for.
```

Comment:
```suggestion
many environments for which
.Fx
supports compilation.
```

Reply:
No

Reply:
"This Is the Sort of Nonsense Up With Which I Will Not Put"
or some other poorly attested quote.
Ending a sentence with a preposition is 100% valid English, so says my copy-editor English-professor wife.